<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWindPat</class>
 <widget class="QWidget" name="SWindPat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="HelpButton" name="b_help">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../Resources/pyleecan.qrc">:/images/images/icon/help_16.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_wind_type">
       <item>
        <property name="text">
         <string>Double Layer Concentrated Orthoradial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Layer Concentrated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double Layer Distributed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Layer Distributed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double Layer Concentrated Radial</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="in_qs">
       <property name="text">
        <string>qs :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="si_qs"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="img_wind_pat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../Resources/.....qrc">:/images/images/MachineSetup/WindingPattern/Type_Wind_6.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="in_coil_pitch">
           <property name="text">
            <string>coil_pitch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="si_coil_pitch"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="in_Nslot">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Nslot_shift :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="si_Nslot">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="is_reverse">
         <property name="text">
          <string>reverse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="out_shape">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Winding Matrix Shape : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="out_ms">
          <property name="text">
           <string>ms = Zs / (2*p*qs) = ?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="out_Nperw">
          <property name="toolTip">
           <string>Winding periodicity</string>
          </property>
          <property name="text">
           <string>Nperw :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_preview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpButton</class>
   <extends>QLabel</extends>
   <header>pyleecan/GUI/Tools/HelpButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
